android/cli/lib/gradle-wrapper.js

```diff
@@ -379,6 +406,14 @@ async function writeJavaPropertiesFile(filePath, properties) {
 
 	// Create the properties files with the text lines generated above.
 	await fs.writeFile(filePath, fileLines.join('\n') + '\n');
+
+	// Add own configuration is provided. Following a proper format is on own risk!
+	const fileName = filePath.split('/').pop();
+	const gradlePropertiesOverwriteFile = path.join(platformPath, fileName);
+
+	if (fs.existsSync(gradlePropertiesOverwriteFile)) {
```

Comment:
if we're being consistent about using async/await:
`if (await fs.exists(gradlePropertiesOverwriteFile)) {`